pull_infra.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+
+def run(cmd, cwd=None, check=True):
+    print(f"🔧 Running: {' '.join(cmd)}")
+    result = subprocess.run(cmd, cwd=cwd, capture_output=True, text=True)
+    if check and result.returncode != 0:
+        print(f"❌ Error: {result.stderr.strip()}")
+        sys.exit(result.returncode)
+    return result.stdout.strip()
+
+
+def vendor_branch_exists(branch_name):
+    return branch_name in run(["git", "branch"], check=False)
+
+
+def vendor_remote_exists(remote_name):
+    return remote_name in run(["git", "remote"], check=False)
```

Comment:
```suggestion
    return remote_name in run(["git", "remote", "--verbose"], check=False)
```

Similar problem as for the `vendor_branch_exists` function, though it's git, so the flags aren't spelled consistently 🤷‍♂️ .

See:
https://git-scm.com/docs/git-remote

Reply:
verbose will add the url in the list output and I just need the remote name